src/main/java/github/scarsz/discordsrv/commands/CommandBroadcast.java

```diff
@@ -45,7 +45,7 @@ public static void execute(CommandSender sender, String[] args) {
         TextChannel target = null;
 
         if (args.length > 0 && args[0].startsWith("#")) {
-            String raw = args[0].replace("#", "");
+            String raw = args[0].replace("#", "").replace("\\n", "\n");
```

Comment:
This is only used to get the channel, thus it isn't neccasery to replace newlines here

Reply:
Changed